beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/hive/HiveClient.java

```diff
@@ -105,6 +108,17 @@ public boolean tableExists(String databaseName, String tableName) {
     }
   }
 
+  @Override
+  public Map<String, String> getTableProperties(String databaseName, String tableName) {
```

Comment:
Please just check for null and return an empty map if so. A table can have a null return as getParameters best to nip this in the bud here so all calling code can freely call map.get without NPE